string name
OpenHRP_AbsoluteForceSensorService_forcemomentOffsetParam i_param
---
bool operation_return
